apps/mobile/src/components/InsideRouter/atoms.ts

```diff
@@ -32,6 +51,7 @@ export const refreshBtcPriceActionAtom = atom<undefined, [], Task<boolean>>(
           return false
         },
         (r) => {
+          set(btcPriceLastUpdateAtAtom, UnixMilliseconds.parse(Date.now()))
```

Comment:
FYI there is already method for this: `unixMillisecondsNow` in `@vexl-next/domain/dist/utility/UnixMilliseconds.brand`